### ISSUE TITLE
Learn Layout section: Media Queries

### ISSUE THREAD

**Issue by rachelandrew** [mdn/sprints#1964]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [*] New documentation
- [ ] Correction or update

# Details

Add a lesson in the learn layout section covering media queries and approaches to using them, for example mobile first design.

**Comment by rachelandrew** [comment]
@chrisdavidmills here is a first draft of the media queries article https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Using_Media_Queries

**Comment by chrisdavidmills** [comment]
@rachelandrew thanks Rachel; this is looking really good. A couple of bits of feedback:

* Where do you think we should put it in the series? After the responsive design article?
* The use cases for width/height are really obvious, but it might be good to write a few words about exactly what you'd use orientation/hover/pointer for — a couple of specific use case descriptions would be nice.
* I've moved it to here — https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Media_queries — and changed the title. I wanted to make sure it is differentiated easily from the existing "Using media queries" article in the main MDN.

**Comment by rachelandrew** [comment]
Yeah I think after the responsive article as that introduces the general RWD ideas, of which media queries is one.

I have added some use case examples as requested.

**Comment by chrisdavidmills** [comment]
OK, I've added it to the CSS layout landing page, fixed the previous/next navigation on the relevant pages so the ordering works, and added it to the "In this module" links at the bottom of each article in the series.

I've also submitted a PR to add it to the sidebar: https://github.com/mdn/kumascript/pull/1234
